client/src/containers/analysis-sidebar/component.tsx

```diff
@@ -141,6 +146,7 @@ const ScenariosComponent: React.FC<{ scrollref?: MutableRefObject<HTMLDivElement
                   className="block w-full"
                   variant="primary"
                   size="xl"
+                  disabled={!canCreateScenario}
```

Comment:
Right now we are not assigning any permission by default, should I quickly implement this on our side? It should be fairly quick happy to do so, let me know!

Reply:
Yes, I think so. The roles have already permissions linked. So, they should be there. As we agreed, we will base the front end on the permissions we receive instead of asume the permissions by the role.

Reply:
Fair enough. On it!
Are we still aligned on our previous agreement that a admin role bypasses all permission checks?

Reply:
Yes, but in other way. We'll allow unless permissions say no.